layouts/partials/sidebar-tree.html

```diff
@@ -8,7 +8,7 @@
     </button>
   </form>
   {{ end }}
-  <nav class="collapse td-sidebar-nav pt-2 pl-4" id="td-section-nav">
+  <nav class="collapse td-sidebar-nav" id="td-section-nav">
```

Comment:
I don't understand the semantics of this change; can you clarify it @Arhell?

Reply:
This is done so that when closing the dropdown there would not be a small “jumping”, to remove it you need to remove all the paddings and margins from the wrapper, which is done, and indents were added to the internal elements

Reply:
Ah, so the class `pt-2` is used (only) to set a CSS `padding-top`, and `pl-4` sets CSS `padding-left`? If I've guessed correctly please confirm; that would explain why it's OK to update the HTML here.

Reply:
Yes that's right

Reply:
For what this PR is trying to solve, the changes seem OK.
The biggest difference I see is the the hover interaction. I don't notice the "jump".
/lgtm